c/kastore.h

```diff
@@ -427,12 +432,14 @@ int kastore_gets_float64(
 @rst
 A key with the specified length is inserted into the store and associated with
 an array of the specified type and number of elements. The contents of the
-specified key and array are copied. Keys can be any sequence of bytes but must
-be at least one byte long and be unique. There is no restriction on the
-contents of arrays. This is the most general form of ``put`` operation in
-kastore; when the type of the array is known and the keys are standard C
-strings, it is usually more convenient to use the :ref:`typed variants
-<sec_c_api_typed_put>` of this function.
+specified key and array are copied unless KAS_BORROWS_ARRAY is specified.
```

Comment:
might be more obvious to say "unless the KAS_BORROWS_ARRAY flag is specified" - there's no indiction of what KAS_BORROWS_ARRAY is here